### ISSUE TITLE
Works but getting the error output below on every invocation

### ISSUE THREAD

**Issue by luckman212** [hdb/fastmask#1]
I installed fastmask today in a new venv using Python 3.12.3 on macOS 14.4.1

It works, but I'm getting the error output below on every invocation:

```
$ fastmask list --active
Python-dotenv could not parse statement starting at line 1
Python-dotenv could not parse statement starting at line 17
Python-dotenv could not parse statement starting at line 18
Python-dotenv could not parse statement starting at line 19
Python-dotenv could not parse statement starting at line 20
Python-dotenv could not parse statement starting at line 21
Python-dotenv could not parse statement starting at line 22
Python-dotenv could not parse statement starting at line 23
```

Is it required that I create an `.env` file somewhere to store account, apikey etc?

**Comment by hdb** [comment]
Is it possible you have a `.env` file somewhere in a parent directory that python-dotenv is unable parse? If that is indeed the issue then I can add an option to skip the `.env` parsing.

But no, the `.env` file is not required - it should work as long as the environment variables are set for account and API key.

**Comment by luckman212** [comment]
Thanks - no I don't have any `.env` files in the current or any parent dir. As a test I changed to the `/` dir just in case. Still got the errors. Strange...

**Comment by hdb** [comment]
I'm still not sure of the source of the error, but in any case I added an option to skip parsing `.env` by setting the environment variable `SKIP_PYTHONDOTENV=1`.

**Comment by hdb** [comment]
I think this is resolved in 92e9dcfaac7d12abf71edb5fb499d49e0836f7a5 so I am going to close this, but feel free to re-open if it is not working as expected. There is an [open upstream issue](https://github.com/theskumar/python-dotenv/issues/510) requesting this feature, though it doesn't look like there is currently a plan to add a standard method of disabling python-dotenv `.env` parsing.

**Comment by hdb** [comment]
Reverted with 99fa622b30c1f6395ae54dd0a9879592691d9b71 as this is now implemented upstream with the `PYTHON_DOTENV_DISABLED` flag.
